website/app.js

```diff
@@ -154,7 +154,7 @@ function generate(
     size: {
       height: 300,
       // @ts-ignore
-      width: window.chartWidth || 375 // TODO: do not use global variable
+      width: document.title === "deno" ? 375 : 800
```

Comment:
I don't think it's a good idea to change the layout based on title. We need a more principled approach.